modules/imgproc/src/intersection.cpp

```diff
@@ -114,10 +122,20 @@ static int _rotatedRectangleIntersection( const RotatedRect& rect1, const Rotate
             float vx2 = vec2[j].x;
             float vy2 = vec2[j].y;
 
+            float numericalScalingFactor = std::min(std::abs(vx1*vx1+vy1*vy1), std::abs(vx2*vx2+vy2*vy2));
+            numericalScalingFactor = !numericalScalingFactor ? 1.f : 1.f/(numericalScalingFactor);
+            if (std::isinf(numericalScalingFactor))
+                numericalScalingFactor = 1.f;
```

Comment:
Other OpenCV code uses this pattern without secondary check:
```
numericalScalingFactor = std::abs(numericalScalingFactor) < 1e-6 ? 1.f : 1.f/(numericalScalingFactor);
```

`std::abs` call may be excluded too as we have abs call above.

---

Could you please to rename variable numerical -> normalize? e.g. `normalizationScale` or similar

Reply:
:+1: